### ISSUE TITLE
Getting started page

### ISSUE THREAD

**Issue by nijel** [WeblateOrg/weblate#2789]
**Is your feature request related to a problem? Please describe.**
When some project starts using Weblate, the admins are typically not aware of all features. The getting started page should guide them trough possible customizations and options to give good guidance.

**Describe the solution you'd like**
Just after creating component (and completing import progress) getting started page would be shown with things which might be good to configure.

* GitHub/GitLab/Bitbucket/... integration
* Translation instructions
* Language consistency, see https://github.com/WeblateOrg/weblate/issues/2811
* Other addons to install (eg. LINGUAS update in case the file is present)

[GitHub community](https://github.com/WeblateOrg/weblate/community) might be good inspiration.

**Comment by comradekingu** [comment]
It would be really nice if this was displayed in some visual manner for all to see.
It could also be used to gauge the sorting in the recommended project tab, and to find dormant projects.

Brainstorming some criteria

- Added license
- Screenshots for % of strings
- Linking set up
- Direct editing 
- Instructions for upstream contribution in place
- Votes for how friendly upstream project is to translations related effort
- TM turned on
- Source checks / colour-coding how many of them fail.
- Announcement made
- Localization.txt in source repo linking to Weblate
- Release cadence info. (could be automated)
- Time-frames set up, detailing string freeze etc.
- Undergone review to make sure the source strings are in good shape.

**Comment by ccppoo** [comment]
With short cut keys (like Tab -> enter to save) instructions

**Comment by comradekingu** [comment]
@ccppoo Could you explain that?

**Comment by ccppoo** [comment]
@comradekingu  
https://docs.weblate.org/en/latest/user/translating.html#keyboard-shortcuts
https://github.com/WeblateOrg/weblate/issues/403

It will be useful for who don't read manuals at first
including me haha..

**Comment by comradekingu** [comment]
@nijel I think a point about considering to turn off language consistency is in order, as in my head it only makes sense in very few cases to do so.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).

**Comment by nijel** [comment]
This is how it looks now:

![Screenshot_2019-10-01 Weblate Test](https://user-images.githubusercontent.com/212189/65975330-c12d0780-e46e-11e9-8b34-b619daeed47e.png)

Feedback is welcome, string improvements for strings introduced in  3620a17 from @comradekingu as well :-)

**Comment by comradekingu** [comment]
@nijel Nice, the bottom ones look to be non-english. I'll try to use it once 4.0 lands. Some minor improvements that I can see.

**Comment by nijel** [comment]
The non English ones are descriptions of existing addons. Sorry for posting screenshot in Czech, I didn't realize that. English one is now in the docs: https://docs.weblate.org/en/latest/devel/community.html
